docs_zh_CN/conf.py

```diff
@@ -0,0 +1,197 @@
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+
+from m2r import MdInclude
+from recommonmark.transform import AutoStructify
+
+sys.path.insert(0, os.path.abspath('..'))
+
+version_file = '../mmcv/version.py'
+with open(version_file, 'r') as f:
+    exec(compile(f.read(), version_file, 'exec'))
+__version__ = locals()['__version__']
+
+# -- Project information -----------------------------------------------------
+
+project = 'mmcv'
+copyright = '2018-2019, Kai Chen'
```

Comment:
How about changing it to OpenMMLab authors. see examples in other repo

Reply:
also in docs/conf.py

Reply:
got it